### PR TITLE
npm support

### PR DESCRIPTION
**Pull request by brrd**
This PR adds npm compatibility (#7).

I used version "1.0.0" in package.json because I couldn't find the actual version in the source code. I can change it if required.

Remaining TODOs are:

- [x] ~~Add the `version` in bower.json.~~ -> `version` is deprecated in bower.json
- [ ] Publish the module on npm.

I believe it is better if the actual developer is also the maintainer of the npm package. Most npm packages are not up-to-date when this is not the case. @adamschwartz would you accept to create an account on npmjs.com and `npm publish` each new version of chrome-tabs?

### CODE REVIEW

**Comment by adamschwartz** [comment]
This is awesome. Thanks  @brrd. I’d feel more comfortable with `0.1.0` since personally I don’t see this is `1.0.0` worthy just yet, not that it matters to me that much. Thanks again!

File: bower.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
     "bower_components",
     "test",
     "tests"
-  ]
+  ],
+  "dependencies": {
+    "draggabilly": "^2.1.1"
```
Comment (adamschwartz) [inline bower.json]: I’d prefer this be locked to `2.1.1`.

File: index.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
       <script async="true" id="_carbonads_js" src="//cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=adamschwartzco"></script>
     </div>
 
-    <script src="js/deps/draggabilly.js"></script>
+    <script src="https://unpkg.com/draggabilly@2/dist/draggabilly.pkgd.min.js"></script>
```
Comment (adamschwartz) [inline index.html]: And this locked to `https://unpkg.com/draggabilly@2.1.1/dist/draggabilly.pkgd.min.js`.

**Comment by brrd** [comment]
Done: draggabilly@2.1.1 is used everywhere.

**Comment by adamschwartz** [comment]
Thanks for making those changes.

Hmm, perhaps we can persuade @alanshaw to allow us to change the `chrome-tabs` npm package name to point to this repo? (I assume that’s allowed?)

Otherwise I guess we’ll have to pick a new name.

**Comment by brrd** [comment]
> Hmm, perhaps we can persuade @alanshaw to allow us to change the chrome-tabs npm package name to point to this repo? (I assume that’s allowed?)

This is possible, but not that simple: https://docs.npmjs.com/misc/disputes
You need to email @alanshaw ([email adress here](https://www.npmjs.com/~alanshaw)) with CC support@npmjs.com and wait for an answer.

And renaming the current npm package would maybe cause issues to developer who already use @alanshaw module as a dependency (though you can easily fix this by incrementing the version number).

In my opinion `chrome-tabs`, `@adamschwartz/chrome-tabs`, `chrome-tabs-js` or any other name would be the same for developers as soon as the way to install it with npm is clearly documented somewhere.  I think the most important is to make this great module available on npm!

**Comment by adamschwartz** [comment]
Given that his repo is a fork of this one, and that we have a willing person (you, @brrd) to do what it takes to get this on there, it seems likely he’d be receptive to making the change. Would you mind reaching out? If you don’t hear back within a week then we’ll name it `chrome-tabs-js` or something like that and that will be that. Thanks for looking into this!

**Comment by brrd** [comment]
Sounds good. I will create an issue in https://github.com/alanshaw/br-chrome-tabs to keep the conversation open.

**Comment by brrd** [comment]
The npm team suggested to publish the first version as v.1.0.0 in order to avoid conflicts with the previous chrome-tabs module. Do you agree @adamschwartz?

**Comment by alanshaw** [comment]
To anyone reading this using the pre 1.0 version, it has moved to https://www.npmjs.com/package/br-chrome-tabs

**Comment by adamschwartz** [comment]
@brrd Sounds good, thanks.

**Comment by adamschwartz** [comment]
I went ahead and made the relevant changes. https://github.com/adamschwartz/chrome-tabs/commit/c116a0f774b985ebb8da102c034e29498259b752 !

**Pull request closed without merging**